ios/Flutter/Flutter.podspec

```diff
@@ -1,18 +0,0 @@
-#
-# NOTE: This podspec is NOT to be published. It is only used as a local source!
-#       This is a generated file; do not edit or check into version control.
-#
-
-Pod::Spec.new do |s|
```

Comment:
Why is it being deleted?

Reply:
I must have messed up with .gitignore. I have added it to the new commit.